third_party/sycl/crosstool/computecpp.tpl

```diff
@@ -12,6 +12,7 @@ CURRENT_DIR = os.path.dirname(sys.argv[0])
 COMPUTECPP_ROOT = CURRENT_DIR + '/../sycl/'
 COMPUTECPP_DRIVER= COMPUTECPP_ROOT + 'bin/compute++'
 COMPUTECPP_INCLUDE = COMPUTECPP_ROOT + 'include'
+SYCL_NO_DOUBLE = os.getenv('SYCL_NO_DOUBLE')
```

Comment:
It's better not to use environment variables since it makes it harder to debug. Prefer to use the bazel.rc below.